keylime/cloud_verifier_tornado.py

```diff
@@ -1658,19 +1658,25 @@ def post(self) -> None:
             logger.warning("POST returning 400 response. missing query parameter 'data'")
             return
 
+        attestation_response: Dict[str, Any] = {}
+
+        attestation_response["valid"] = False
+        attestation_response["claims"] = {}
+        attestation_response["failures"] = []
+
         try:
             if evidence_type == "tpm":
-                attestation_failure = self._tpm_verify(data)
-            elif evidence_type == "snp":
-                attestation_failure = self._sev_snp_verify(data)
+                (claims, attestation_failure) = self._tpm_verify(data)
+                attestation_response["claims"] = claims
+            elif evidence_type == "tee":
+                (claims, attestation_failure) = self._tee_verify(data)
```

Comment:
Here we set the claims and return it unconditionally before checking if `attestation_failure` exists further on. We should probably return the claims only on successful verification, like we do in `verify_attestation()` (`snp.py`)